exercism/hamming/hamming.src.js

```diff
@@ -0,0 +1,17 @@
+var Hamming = function() {};
+
+Hamming.prototype.compute =  function (a, b) {
```

Comment:
first off your formatting is making this function almost impossible to read and therefore reason about. This would be proper formatting:

```js

Hamming.prototype.compute =  function (a, b) {
  if (a.length === b.length) {
    var diff = 0;
    for (var i = 0; i <= a.length; i++) {
      if (a[i] != b[i]) {
        diff++;            
      };
    };
    return diff;
  } else {
    throw new Error('DNA strands must be of equal length.');
  };
};
```